Cargo.toml

```diff
@@ -14,6 +14,7 @@ version = "0.5.1"
 bitflags = "^1.0"
 log = "~0.4"
 parking_lot = "^0.5"
+rand = "^0.4"
```

Comment:
This can be marked as an optional dependency, and then enabled under the `voice` feature.